Cargo.toml

```diff
@@ -28,7 +28,8 @@ members = [
   "serverless",
   "bin_tests",
   "data-pipeline",
-  "data-pipeline-ffi"
+  "data-pipeline-ffi",
+  "ddsketch",
```

Comment:
What's the impact on the size of the binary? (I should add an automatic comment for any modif in that file :p)

Reply:
The release artifacts are 293,2MB and the last build on main is 290,8MB